cmd/run/cmd.go

```diff
@@ -0,0 +1,332 @@
+package run
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	buildahCopiah "github.com/containers/buildah/copier"
+	"github.com/containers/podman/v4/libpod/define"
+	"github.com/containers/podman/v4/pkg/bindings"
+	"github.com/containers/podman/v4/pkg/bindings/containers"
+	"github.com/containers/podman/v4/pkg/errorhandling"
+	"github.com/containers/podman/v4/pkg/specgen"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	initCmd "github.com/openshift/occ/cmd/init"
+	"github.com/openshift/occ/pkg/config"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+	"runtime"
+	"strings"
+)
+
+var (
+	exec               string
+	tag                string
+	disableConsolePort bool
+)
+
+func NewRunCmd() *cobra.Command {
+	var runCmd = &cobra.Command{
+		Use:   "run [cluster_id]",
+		Short: "Runs an OCM container instance",
+		Long: `Run will start up an OCM container instance using a given configuration file.
+This will default to "~/.config/occ/config.yaml" if one isn't provided.`,
```

Comment:
I think we can leave off the config default path here, since this is configurable.  If the config file doesn't exist we should exit.